projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -955,7 +955,7 @@ function get_paywall_blocks( $newsletter_access_level ) {
 	} else {
 		if ( ( new Host() )->is_wpcom_simple() ) {
 			// custom domain
-			$sign_in_link = wpcom_logmein_redirect_url( get_current_url(), false, null, 'link' );
+			$sign_in_link = wpcom_logmein_redirect_url( get_current_url(), false, get_current_url(), 'link', get_current_blog_id() );
```

Comment:
we don't need to pass the second `get_current_url()` anymore,right?

Reply:
Correct. Will remove shortly.